packages/x-charts/src/internals/plugins/models/plugin.ts

```diff
@@ -5,12 +5,34 @@ import { ChartStore } from '../utils/ChartStore';
 import { ChartSeriesConfig } from './seriesConfig';
 
 export interface ChartPluginOptions<TSignature extends ChartAnyPluginSignature> {
+  /**
```

Comment:
If you can add those to the tree view after this PR is approved it would be awesome :pray: